test/Microsoft.Health.Fhir.Shared.Tests.Integration/Features/Operations/Import/SqlServerFhirDataBulkOperationTests.cs

```diff
@@ -268,17 +268,17 @@ public async Task GivenUnclusteredIndexes_WhenRebuildDisableIndexes_ThenIndexSho
                 isDisabled = await GetIndexDisableStatus(index.indexName);
                 Assert.True(isDisabled);
 
-                await RebuildIndex(index.tableName, index.indexName, index.pageCompression);
-                isDisabled = await GetIndexDisableStatus(index.indexName);
-                Assert.False(isDisabled);
+                ////await RebuildIndex(index.tableName, index.indexName, index.pageCompression);
```

Comment:
Can we just remove these lines if they aren't needed anymore?

Reply:
I will let Tong to comment.

Reply:
We can remove old rebuild indexes SProc related code, they are not needed any more.